src/ElvClient.js

```diff
@@ -967,10 +970,17 @@ class ElvClient {
    * Create a signature for the specified string
    *
    * @param {string} string - The string to sign
+   * @param {boolean} usePersonal - use EIP-191 personal_sign for signing
```

Comment:
please change to `@param {boolean=} usePersonal=false` to indicate it is optional and defaults to false